src/test/unit_tests/test_nc_cli.js

```diff
@@ -991,16 +991,25 @@ mocha.describe('manage_nsfs cli', function() {
             assert_whitelist(config_data, config_options);
         });
 
-        mocha.it('cli whitelist ips is empty', async function() {
+        mocha.it('should fail - cli whitelist ips is empty', async function() {
```

Comment:
I would expand the `it` and add a comment so it will be clear.
```suggestion
        mocha.it('should fail - cli whitelist ips is an empty string', async function() {
        // we allow unset the ips list with empty array '[]' and not an empty string ''
```